### ISSUE TITLE
UserWarning: Using a target size that is different to the input size

### ISSUE THREAD

**Issue by thotd21** [switchablenorms/DeepFashion_Try_On#31]
When running the train only see the pytorch warning log about the input image size? I also use VITON Dataset.
Is this the problem that causes the output quality error as shown below:
![tho](https://user-images.githubusercontent.com/22317021/88612937-958ea400-d0b6-11ea-8a40-67e26549d5d7.jpg)

> /usr/local/lib/python3.6/dist-packages/torchvision/transforms/transforms.py:211: UserWarning: The use of the transforms.Scale transform is deprecated, please use transforms.Resize instead.
>   "please use transforms.Resize instead.")
> /usr/local/lib/python3.6/dist-packages/torchvision/transforms/transforms.py:211: UserWarning: The use of the transforms.Scale transform is deprecated, please use transforms.Resize instead.
>   "please use transforms.Resize instead.")
> /usr/local/lib/python3.6/dist-packages/torch/nn/_reduction.py:43: UserWarning: size_average and reduce args will be deprecated, please use reduction='mean' instead.
>   warnings.warn(warning.format(ret))
> /usr/local/lib/python3.6/dist-packages/torch/nn/functional.py:1558: UserWarning: nn.functional.tanh is deprecated. Use torch.tanh instead.
>   warnings.warn("nn.functional.tanh is deprecated. Use torch.tanh instead.")
> /usr/local/lib/python3.6/dist-packages/torch/nn/functional.py:3226: UserWarning: Default grid_sample and affine_grid behavior has changed to align_corners=False since 1.3.0. Please specify align_corners=True if the old behavior is desired. See the documentation of grid_sample for details.
>   warnings.warn("Default grid_sample and affine_grid behavior has changed "
> /pytorch/torch/csrc/utils/python_arg_parser.cpp:756: UserWarning: This overload of nonzero is deprecated:
> 	nonzero(Tensor input, *, Tensor out)
> Consider using one of the following signatures instead:
> 	nonzero(Tensor input, *, bool as_tuple)
> /usr/local/lib/python3.6/dist-packages/torch/nn/modules/loss.py:88: UserWarning: Using a target size (torch.Size([2, 1, 256, 192])) that is different to the input size (torch.Size([2, 256, 192])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
>   return F.l1_loss(input, target, reduction=self.reduction)

Runtime environment is:

> Cuda compilation tools, release 10.1, V10.1.243
> torch.__version__: 1.5.1+cu101

**Comment by SayleeMore** [comment]
@thotd21  Hey Can you help me regarding this? Can you please contact me at sayleevmore3@gmail.com

**Comment by chethan-reddy2** [comment]
Hello @thotd21 did you solve the error?

**Comment by josearangos** [comment]
@thotd21 thotd21 How do I run with custom images? How to create the images that are inside the test_label folder?

**Comment by HAN-YANG-HIT** [comment]
u can use the colab.
